corehq/apps/receiverwrapper/tests/data/form_data_with_non_bmp_chars.xml

```diff
@@ -0,0 +1,89 @@
+        7 U+01D4CC ‹𝓌› GC=Ll    MATHEMATICAL SCRIPT SMALL W
+        7 U+01D516 ‹𝔖› GC=Lu    MATHEMATICAL FRAKTUR CAPITAL S
+        7 U+01D4BE ‹𝒾› GC=Ll    MATHEMATICAL SCRIPT SMALL I
+        7 U+01D4CC ‹𝓌› GC=Ll    MATHEMATICAL SCRIPT SMALL W
+        7 U+01D516 ‹𝔖› GC=Lu    MATHEMATICAL FRAKTUR CAPITAL S
+        4 U+01D4CF ‹𝓏› GC=Ll    MATHEMATICAL SCRIPT SMALL Z
+        4 U+01D53B ‹𝔻› GC=Lu    MATHEMATICAL DOUBLE-STRUCK CAPITAL D
+        4 U+01D54B ‹𝕋› GC=Lu    MATHEMATICAL DOUBLE-STRUCK CAPITAL T
+        3 U+01D4BB ‹𝒻› GC=Ll    MATHEMATICAL SCRIPT SMALL F
+        3 U+01D4CA ‹𝓊› GC=Ll    MATHEMATICAL SCRIPT SMALL U
+        3 U+01D507 ‹𝔇› GC=Lu    MATHEMATICAL FRAKTUR CAPITAL D
+        3 U+01D542 ‹𝕂› GC=Lu    MATHEMATICAL DOUBLE-STRUCK CAPITAL K
+        3 U+01D546 ‹𝕆› GC=Lu    MATHEMATICAL DOUBLE-STRUCK CAPITAL O
+        2 U+01D4BD ‹𝒽› GC=Ll    MATHEMATICAL SCRIPT SMALL H
+        2 U+01D4C5 ‹𝓅› GC=Ll    MATHEMATICAL SCRIPT SMALL P
+        2 U+01D505 ‹𝔅› GC=Lu    MATHEMATICAL FRAKTUR CAPITAL B
+        2 U+01D50E ‹𝔎› GC=Lu    MATHEMATICAL FRAKTUR CAPITAL K
+        2 U+01D541 ‹𝕁› GC=Lu    MATHEMATICAL DOUBLE-STRUCK CAPITAL J
+        2 U+01D543 ‹𝕃› GC=Lu    MATHEMATICAL DOUBLE-STRUCK CAPITAL L
+        2 U+100002 ‹􀀂› GC=Co    Private
+        1 U+01D4B8 ‹𝒸› GC=Ll    MATHEMATICAL SCRIPT SMALL C
+        1 U+01D4C1 ‹𝓁› GC=Ll    MATHEMATICAL SCRIPT SMALL L
+        1 U+01D53D ‹𝔽› GC=Lu    MATHEMATICAL DOUBLE-STRUCK CAPITAL F
+        1 U+01D53E ‹𝔾› GC=Lu    MATHEMATICAL DOUBLE-STRUCK CAPITAL G
+        1 U+01D54C ‹𝕌› GC=Lu    MATHEMATICAL DOUBLE-STRUCK CAPITAL U
+        1 U+01D6A4 ‹𝚤› GC=Ll    MATHEMATICAL ITALIC SMALL DOTLESS I
+        1 U+01D7D9 ‹𝟙› GC=Nd    MATHEMATICAL DOUBLE-STRUCK DIGIT ONE
+         😀 😁 😂 🤣 😃 😄 😅 😆 😉 😊 😋 😎 😍 😘 🥰 😗 😙 😚 ☺️ 🙂 🤗 🤩 🤔 🤨 😐 😑 😶 🙄 😏 😣 😥 😮 🤐 😯 😪 😫 😴 😌 😛 😜 😝 🤤 😒 😓 😔 😕 🙃 🤑 😲 ☹️ 🙁 😖 😞 😟 😤 😢 😭 😦 😧 😨 😩 🤯 😬 😰 😱 🥵 🥶 😳 🤪 😵 😡 😠 🤬 😷 🤒 🤕 🤢 🤮 🤧 😇 🤠 🤡 🥳 🥴 🥺 🤥 🤫 🤭 🧐 🤓 😈 👿 👹 👺 💀 👻 👽 🤖 💩 😺 😸 😹 😻 😼 😽 🙀 😿 😾
+
+        People and Fantasy
+        👶 👧 🧒 👦 👩 🧑 👨 👵 🧓 👴 👲 👳‍♀️ 👳‍♂️ 🧕 🧔 👱‍♂️ 👱‍♀️ 👨‍🦰 👩‍🦰 👨‍🦱 👩‍🦱 👨‍🦲 👩‍🦲 👨‍🦳 👩‍🦳 🦸‍♀️ 🦸‍♂️ 🦹‍♀️ 🦹‍♂️ 👮‍♀️ 👮‍♂️ 👷‍♀️ 👷‍♂️ 💂‍♀️ 💂‍♂️ 🕵️‍♀️ 🕵️‍♂️ 👩‍⚕️ 👨‍⚕️ 👩‍🌾 👨‍🌾 👩‍🍳 👨‍🍳 👩‍🎓 👨‍🎓 👩‍🎤 👨‍🎤 👩‍🏫 👨‍🏫 👩‍🏭 👨‍🏭 👩‍💻 👨‍💻 👩‍💼 👨‍💼 👩‍🔧 👨‍🔧 👩‍🔬 👨‍🔬 👩‍🎨 👨‍🎨 👩‍🚒 👨‍🚒 👩‍✈️ 👨‍✈️ 👩‍🚀 👨‍🚀 👩‍⚖️ 👨‍⚖️ 👰 🤵 👸 🤴 🤶 🎅 🧙‍♀️ 🧙‍♂️ 🧝‍♀️ 🧝‍♂️ 🧛‍♀️ 🧛‍♂️ 🧟‍♀️ 🧟‍♂️ 🧞‍♀️ 🧞‍♂️ 🧜‍♀️ 🧜‍♂️ 🧚‍♀️ 🧚‍♂️ 👼 🤰 🤱 🙇‍♀️ 🙇‍♂️ 💁‍♀️ 💁‍♂️ 🙅‍♀️ 🙅‍♂️ 🙆‍♀️ 🙆‍♂️ 🙋‍♀️ 🙋‍♂️ 🤦‍♀️ 🤦‍♂️ 🤷‍♀️ 🤷‍♂️ 🙎‍♀️ 🙎‍♂️ 🙍‍♀️ 🙍‍♂️ 💇‍♀️ 💇‍♂️ 💆‍♀️ 💆‍♂️ 🧖‍♀️ 🧖‍♂️ 💅 🤳 💃 🕺 👯‍♀️ 👯‍♂️ 🕴 🚶‍♀️ 🚶‍♂️ 🏃‍♀️ 🏃‍♂️ 👫 👭 👬 💑 👩‍❤️‍👩 👨‍❤️‍👨 💏 👩‍❤️‍💋‍👩 👨‍❤️‍💋‍👨 👪 👨‍👩‍👧 👨‍👩‍👧‍👦 👨‍👩‍👦‍👦 👨‍👩‍👧‍👧 👩‍👩‍👦 👩‍👩‍👧 👩‍👩‍👧‍👦 👩‍👩‍👦‍👦 👩‍👩‍👧‍👧 👨‍👨‍👦 👨‍👨‍👧 👨‍👨‍👧‍👦 👨‍👨‍👦‍👦 👨‍👨‍👧‍👧 👩‍👦 👩‍👧 👩‍👧‍👦 👩‍👦‍👦 👩‍👧‍👧 👨‍👦 👨‍👧 👨‍👧‍👦 👨‍👦‍👦 👨‍👧‍👧 🤲 👐 🙌 👏 🤝 👍 👎 👊 ✊ 🤛 🤜 🤞 ✌️ 🤟 🤘 👌 👈 👉 👆 👇 ☝️ ✋ 🤚 🖐 🖖 👋 🤙 💪 🦵 🦶 🖕 ✍️ 🙏 💍 💄 💋 👄 👅 👂 👃 👣 👁 👀 🧠 🦴 🦷 🗣 👤 👥
+
+        Clothing and Accessories
+        🧥 👚 👕 👖 👔 👗 👙 👘 👠 👡 👢 👞 👟 🥾 🥿 🧦 🧤 🧣 🎩 🧢 👒 🎓 ⛑ 👑 👝 👛 👜 💼 🎒 👓 🕶 🥽 🥼 🌂 🧵 🧶
+        </what_is_your_name>
```

Comment:
oh wow, I think we should also add a test when adding one or two of these in name field. Also that would help check that that makes it to the case name properly.

Reply:
Done. It seems to be working fine

Reply:
thank you, do we need to check what case name looks like post submission?

Reply:
Yeah. We can definitely do that. That would close the loop. 
Thanks for the suggestion 😄